pylint/__pkginfo__.py

```diff
@@ -27,7 +27,7 @@ def get_numversion_from_version(v: str) -> Tuple:
                 version.append(int(num))
             except ValueError:
                 version.append(0)
-    while len(version) != 3:
+    for _ in range(3 - len(version)):
```

Comment:
IMO the `while` statement is a bit easier to read and understand.

Reply:
I tried another way with no loop at all. Compare this with https://github.com/PyCQA/pylint/pull/4860/files#diff-26d7b9f740481d02edafcd71a3f448d5d2f5a7ff9858bfa59622e72cfc87a3ed